frontend/packages/pipelines-plugin/src/components/pipelines/list-page/PipelineAugmentRunsWrapper.tsx

```diff
@@ -15,7 +15,10 @@ interface PipelineAugmentRunsWrapperProps {
 
 const PipelineAugmentRunsWrapper: React.FC<PipelineAugmentRunsWrapperProps> = (props) => {
   const pipelineData = _.get(props.pipeline, 'data', []);
-  if (pipelineData.length < 1) {
+  if (!props.pipeline.loaded) {
+    return null;
+  }
+  if (props.pipeline.loaded && pipelineData.length < 1) {
```

Comment:
```suggestion
  if (pipelineData.length === 0) {
```
Best be explicit and not mathematically correct 🙂 

Also, don't need to include the inverse of previous boolean statement.